src/fmain.pas

```diff
@@ -6697,106 +6698,123 @@ procedure Tfrmain.milistPropertiesClick(Sender: TObject);
   cbEngineItemText:string='';
   cbQueueItemText:string='';
 
-  deDestinationTextIsSame:integer=0; // 0 - init; 1 - same; 2 - different
-  edtParametersTextIsSame:integer=0;
-  edtPasswordTextIsSame:integer=0;
-  edtUserTextIsSame:integer=0;
-  cbEngineItemTextIsSame:integer=0;
-  cbQueueItemTextIsSame:integer=0;
+  deDestinationTextPropState:TPropState=PropInitValue;
+  edtParametersTextPropState:TPropState=PropInitValue;
+  edtPasswordTextPropState:TPropState=PropInitValue;
+  edtUserTextPropState:TPropState=PropInitValue;
+  cbEngineItemTextPropState:TPropState=PropInitValue;
+  cbQueueItemTextPropState:TPropState=PropInitValue;
 
 begin
   if frmain.lvMain.ItemIndex<>-1 then
   begin
     if frmain.lvMain.Items[frmain.lvMain.ItemIndex].SubItems[columntype] = '0' then
     begin
       //////THIS ORDER IS IMPORTANT/////////
-      frnewdown.edtURL.Text:=frmain.lvMain.Items[frmain.lvMain.ItemIndex].SubItems[columnurl];
-      frnewdown.edtFileName.Text:=frmain.lvMain.Items[frmain.lvMain.ItemIndex].SubItems[columnname];
-      frnewdown.deDestination.Text:=frmain.lvMain.Items[frmain.lvMain.ItemIndex].SubItems[columndestiny];
-      enginereload();
-      frnewdown.cbQueue.Items.Clear;
-      for i:=0 to Length(queues)-1 do
-      begin
-       frnewdown.cbQueue.Items.Add(queuenames[i]);
+      with frmain.lvMain.Items[frmain.lvMain.ItemIndex] do
```

Comment:
All the following changes are _much_ better to be reviewed with "Diff settings" -> "Hide whitespace changes" checked.

Reply:
But the online reviewing/comparing is still very bad here.
Probably the best way is to get (download) a local copy of the updated version and compare is with the old code in some local offline diff tool such as WinMerge or BeyondCompare or something similar.